runtime/gc_vlhgc/ContinuationObjectBufferVLHGC.cpp

```diff
@@ -107,3 +109,34 @@ MM_ContinuationObjectBufferVLHGC::addForOnlyCompactedRegion(MM_EnvironmentBase*
 		}
 	}
 }
+
+void
+MM_ContinuationObjectBufferVLHGC::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+	MM_HeapRegionManager *regionManager = extensions->getHeap()->getHeapRegionManager();
+	MM_HeapRegionDescriptorVLHGC *region = NULL;
+	GC_HeapRegionIteratorVLHGC regionIterator(regionManager);
+
+	/* to make sure that previous pruning phase of continuation list(scanContinuationObjects()) is complete */
+	env->_currentTask->synchronizeGCThreads(env, UNIQUE_ID);
+
+	while (NULL != (region = regionIterator.nextRegion())) {
+		if (region->containsObjects()) {
+			if (NULL != region->getContinuationObjectList()->getHeadOfList()) {
+				if (J9MODRON_HANDLE_NEXT_WORK_UNIT(env)) {
+					omrobjectptr_t object = region->getContinuationObjectList()->getHeadOfList();
+					while (NULL != object) {
+						Assert_MM_true(region->isAddressInRegion(object));
+						omrobjectptr_t next = extensions->accessBarrier->getContinuationLink(object);
+						J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF((J9VMThread *)env->getLanguageVMThread(), object);
```

Comment:
This does not compile for Java 8, 11 or 17: `J9VMJDKINTERNALVMCONTINUATION_VMREF` does not exist in those versions.